core/src/gcg.c

```diff
@@ -561,15 +562,15 @@ gcg_parse_status_t parse_next_gcg_line(GCGParser *gcg_parser) {
 
   if (token == GCG_MOVE_TOKEN || token == GCG_PASS_TOKEN ||
       token == GCG_EXCHANGE_TOKEN) {
-    if (game_history->players[0] == NULL || game_history->players[1] == NULL) {
+    if (!game_history->players[0] || !game_history->players[1]) {
       return GCG_PARSE_STATUS_MOVE_BEFORE_PLAYER;
     }
-    if (game_history->letter_distribution == NULL) {
+    if (!game_history->letter_distribution) {
 
-      if (game_history->lexicon_name == NULL) {
+      if (!game_history->lexicon_name) {
```

Comment:
same NULL v 0 question here

Reply:
Leaving as is for now, might change later.